app/commands/v2/import.rb

```diff
@@ -0,0 +1,98 @@
+module Commands
+  module V2
+    class Import < BaseCommand
+      def call
+        unless UuidValidator.valid?(payload[:content_id])
+          raise CommandError.new(
+          code: 422,
+          error_details: {
+            error: {
+              code: 422,
+              message: "Content id not valid",
+              fields: "content_id",
+            }
+          })
+        end
+
+        delete_all(payload[:content_id])
```

Comment:
I've added this functionality now https://github.com/alphagov/publishing-api/pull/622/commits/a6835cce8e256f7811bef7c91ed3e4e5a72b518a